package.json

```diff
@@ -47,12 +48,18 @@
     "custom scrollbar"
   ],
   "dependencies": {
-    "detect-passive-events": "^1.0.4",
-    "faste": "^1.0.3",
-    "tslib": "^1.0.0"
+    "detect-passive-events": "2.0.3",
```

Comment:
pinning dependency for a library is not a good idea. 
While it can make it a little stable - it can bring way more bytes to the end product.
(my concert is mostly around tslib)

Reply:
Fixed